src/backup_restore.cpp

```diff
@@ -128,10 +128,165 @@ void BackupRestore::backupAndRestoreIpzVpd(types::IPZVpdMap& io_srcVpdMap,
                                            const std::string& i_srcPath,
                                            const std::string& i_dstPath)
 {
-    (void)io_srcVpdMap;
-    (void)io_dstVpdMap;
-    (void)i_srcPath;
-    (void)i_dstPath;
+    if (!m_backupRestoreCfgJsonObj.contains("backupMap") ||
+        !m_backupRestoreCfgJsonObj["backupMap"].is_array())
+    {
+        logging::logMessage(
+            "Backup map not found in the backup and restore config JSON.");
```

Comment:
This is not valid if only second condition fails.

Reply:
Updated